package.json

```diff
@@ -37,6 +37,7 @@
     "is-ci": "^3.0.1",
     "joi": "^17.6.0",
     "jsonwebtoken": "^8.5.1",
+    "moment": "^2.29.3",
```

Comment:
we dont use it in the server side
delete it

Reply:
done